README.md

```diff
@@ -348,7 +348,45 @@ meetingSession.audioVideo.realtimeSubscribeToVolumeIndicator(
 );
 ```
 
-**Use case 11.** Detect the most active speaker. For example, you can enlarge the active speaker's video element if available.
+**Use case 11.** Handle `realtimeSubscribeToVolumeIndicator` updates efficiently.
+
+When subscribed to `realtimeSubscribeToVolumeIndicator`, the values are updated if they are changed, else they will be null. To efficiently implement this API, you can listen separately to each of these values. Consequently, your callback will only handle the value, you are interested in.
+
+- To track just mute:
+```js
+const presentAttendeeId = meetingSession.configuration.credentials.attendeeId;
+
+meetingSession.audioVideo.realtimeSubscribeToVolumeIndicator(
+  presentAttendeeId,
+  (attendeeId, volume, muted, signalStrength) => {
+    if (muted === null) {
+      // no mute update - return
+      return;
+    }
+
+    // mute has changed, update UI
+  }
+);
+```
+
+- To track just signal strength:
```

Comment:
How about combining this into the above code block since `presentAttendeeId` is a duplicate declaration?

```
// To track signal strength changes
meetingSession.audioVideo.realtimeSubscribeToVolumeIndicator(
  presentAttendeeId,
  (attendeeId, volume, muted, signalStrength) => {
    if (signalStrength === null) {
      return;
    }
    // The signal strength for "presentAttendeeId" has changed.
  }
);
```

Reply:
Yes makes sense, updated into a single code block.